pep-0013.rst

```diff
@@ -286,6 +286,13 @@ will. While someone is in inactive status, though, they lose their
 active privileges like voting or nominating for the steering council,
 and commit access.
 
+One month before each election, and automated script will be
+run to produce a list of potentially inactive core team members.
+An email will be sent to python-committers mailing list with this list,
+to notify that they've been deemed inactive due to lack of activity
+in the past two months. If they did not respond after two weeks,
+they will be removed from the voter's list.
+
```

Comment:
```suggestion
One month before each election, an automated script will be
run to produce a list of potentially inactive core team members.
An email will be sent to python-committers mailing list with this list,
to notify that they've been deemed inactive due to lack of activity
in the past two years. If they did not respond after two weeks,
they will be removed from the voter's list.
```

I think the current period used by the script is two years rather than two months: https://mail.python.org/archives/list/python-committers@python.org/thread/DLJE25TWAQ2KBGVJUSUO4W7KSZYHFFVC/

Reply:
Oh yes @aeros is correct!  I missed that in my edit.  I'll do another edit and let @Mariatta make the final approval via merge.